src/sentry/integrations/mixins/issues.py

```diff
@@ -348,7 +348,7 @@ def map_external_issues_to_annotations(self, external_issues):
         for ei in external_issues:
             link = self.get_issue_url(ei.key)
             label = self.get_issue_display_name(ei) or ei.key
-            annotations.append(f'<a href="{link}">{label}</a>')
+            annotations.append({"url": link, "display_name": label})
```

Comment:
Already mentioned, but since we changed this to `displayName` in the frontend we need to change it here as well